core/frontend/src/Viewport.ts

```diff
@@ -816,6 +816,21 @@ export abstract class Viewport implements IDisposable {
   public changeBackgroundMapProps(props: BackgroundMapProps): void {
     this.displayStyle.changeBackgroundMapProps(props);
   }
+  /** Get a tile tree ref will load a tree that returns geometry rather than graphics.  Currently only the return of Polyface geometry from reality tile
+   * or map tile trees (with terrain enabled) are supported, calling [[getTileTreeRef]] on other tile trees will return undefined.
+   * Tile geometry can be  by used indirectly by methods such as [[RealityTileTree.drapeLineString]] or by calling [[RealityTileTree.collectTiles]] and
+   * iterating through[[RealityTileCollector.acceptedGeometry]]
+   */
+  public getGeometryTreeRef(modelId?: Id64String): TileTreeReference | undefined {
+    let geometryTreeRef: TileTreeReference | undefined;
+    this.forEachTileTreeRef((treeRef) => {
+      const tree = treeRef.treeOwner.load();
+      if (tree?.modelId === modelId)

```

Comment:
What is supposed to happen if `undefined` is passed for `modelId`? This line will evaluate `true` if `tree` and `modelId` are both undefined.

Reply:
Undefined should not be allowed -- will fix.